pr_check.sh

```diff
@@ -11,30 +11,19 @@ IQE_PLUGINS="rhc"
 IQE_MARKER_EXPRESSION="smoke"
 IQE_FILTER_EXPRESSION="playbook_dispatcher"
 
+
 # Install bonfire repository/initialize
 CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
-curl -s $CICD_URL/bootstrap.sh -o bootstrap.sh
-source bootstrap.sh  # checks out bonfire and changes to "cicd" dir...
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
 
 # Build Playbook Dispatcher image based on the latest commit
-source build.sh
-
-# Deploy the new image to an ephemeral environment
-source _common_deploy_logic.sh
-result=$(bonfire config deploy \
-    --ref-env insights-stage \
-    --app $APP_NAME \
-    --set-template-ref $COMPONENT_NAME=$GIT_COMMIT \
-    --set-template-ref $TEST_SECRETS_COMPONENT_NAME=$GIT_COMMIT \
-    --set-image-tag $IMAGE=$IMAGE_TAG \
-    --get-dependencies)
+source $CICD_ROOT/build.sh
 
-if [ $? -eq 0 ]; then
-    export NAMESPACE=$result
-fi
+# Execute unit tests
+#source $APP_ROOT/unit_test.sh
 
-# Deploy Ingress services to the ephemeral environment
-bonfire config deploy --ref-env insights-stage --app ingress --get-dependencies --namespace $NAMESPACE
```

Comment:
@bsquizz Removing this line will make the Ingress service not be deployed to the ephemeral env and the playbook dispatcher smoke tests have a dependency on that service. What is the best way to have Ingress service deployed together with Playbook Dispatcher?

Reply:
Good catch, I meant to update your template to add ingress under `optionalDependencies` -- that will cause bonfire to deploy ingress when your app is deployed.

Reply:
Fixed